### ISSUE TITLE
Contract and unit tests failing on flake8 error

### ISSUE THREAD

**Issue by pritchyspritch** [alphagov/cyber-security-security-advisory-dashboard#301]
```SKIPPED [1] tests/test_storage.py:26: Breaks when run out of order
SKIPPED [1] tests/test_storage.py:33: Breaks when run out of order
SKIPPED [1] tests/test_storage.py:62: Breaks when run out of order
SKIPPED [1] tests/test_storage.py:43: Breaks when run out of order
FAILED tests/test_app.py::FLAKE8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'
FAILED tests/__init__.py::FLAKE8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'
FAILED tests/test_config.py::FLAKE8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'
FAILED tests/test_splunk.py::FLAKE8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'
FAILED tests/test_storage.py::FLAKE8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'
FAILED tests/test_vulnerable_by_severity_splunk.py::FLAKE8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'
FAILED tests/test_cyber_dependabot.py::FLAKE8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'
FAILED tests/test_audit_lambda.py::FLAKE8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'
FAILED tests/test_vulnerability_summarizer.py::FLAKE8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'```

**Comment by denizgenc** [comment]
From a cursory inspection of the jobs in Concourse that are erroring, one of the things that have changed is the `sec_adv_docker` resource that both of them seem to be retrieving.

Before the error, the sha256 hash of the image began with `c1c58`.

Afterwards, when the errors occur, the hash of the image begins with `d6f38`.

**Comment by Bursade** [comment]
Could anyone find out the root cause of FLAKE8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args' ?
I'm having a similar issue with anothar application.

**Comment by denizgenc** [comment]
This issue seems to arise from line 191 in the `pytest-flake8` plugin, from version `1.0.5` and earlier:

```python
    app.parse_preliminary_options_and_args(args)
```

This was changed in version 1.0.6:

```python
    app = application.Application()
    if not hasattr(app, 'parse_preliminary_options_and_args'):  # flake8 >= 3.8
        ...
```

It turns out that `flake8` version 3.8 no longer has this `parse_preliminary_options_and_args` attribute. Also, flake8 just released version 3.8 on May 11, which explains why our pipeline only recently started erroring: https://pypi.org/project/flake8/

**tl;dr**: Update the version of `pytest-flake8` we're using to 1.0.6 or above.

In our case, we just need to re-run the pipeline, since it will retrieve the most recent version of the plugin automatically.

Related issue in the `pytest-flake8` repo: https://github.com/tholo/pytest-flake8/issues/66

**Comment by denizgenc** [comment]
I ran the pipeline again, and the errors that this issue referred to were resolved (because the pipeline retrieved `pytest-flake8` version 1.0.6).

However, an unrelated error is now popping up - flake8 complains about the following:
```
security-advisory-dashboard-git/tests/test_splunk.py:49:31: F541 f-string is missing placeholders
```

I will be opening a PR for this separately.
